shared/app/nav.native.js

```diff
@@ -365,59 +347,54 @@ class Nav extends Component<Props, {keyboardShowing: boolean}> {
       />
     )
     const layerScreens = this.props.routeStack.filter(r => r.tags && r.tags.layerOnTop)
-    const layers = layerScreens.map(r => r.leafComponent({isActiveRoute: true, shouldRender: true}))
+    const layers = layerScreens.map(r => r.leafComponent({shouldRender: true}))
 
-    // If we have layers, lets add an extra box, else lets just pass through
-    if (layers.count()) {
-      return (
-        <Box style={globalStyles.fillAbsolute}>
-          {shim}
-          {layers}
-        </Box>
-      )
-    } else {
-      return shim
-    }
+    return (
```

Comment:
us switching between having this box and not causes the entire tree to thrash as everything is moving up a level. this is what happens when you open a modal everything would trash cause of this